### PR TITLE
Fixes and clean up

### PR DESCRIPTION
**Pull request by kylon**

### CODE REVIEW

File: destreamer.ts [hunk 1]
```diff
@@ -51,6 +51,13 @@ const argv = yargs.options({
         default: false,
         demandOption: false
     },
+    noThumbnails: {
+        alias: 'nthumb',
```
Comment (lukaarma) [inline destreamer.ts:55]: I wouldn't create an alias for such an option

File: destreamer.ts [hunk 2]
```diff
@@ -61,20 +68,37 @@ const argv = yargs.options({
     }
 }).argv;
 
-function init() {
+async function init() {
+    const isValidUser = !(await isElevated());
+
+    if (!isValidUser) {
+        const usrName = os.platform() === 'win32' ? 'Admin':'root';
```
Comment (lukaarma) [inline destreamer.ts:75]: We can use process.platfort, it's the exact same thing without another module

File: destreamer.ts (hunk 1 above)
Comment (kylon) [reply destreamer.ts:55]: why not? 
full option is lengthy and we are all lazy :P

Comment (snobu) [reply destreamer.ts:53]: I think the cooler name for that would be `--no-art`, but let's rename when we switch to --kebab-case for all CLI args.

**Comment by snobu** [comment]
I have a merge conflict that i'm not sure how to resolve.

```
<<<<<<< tokencache
    if (session == null)
        session = await DoInteractiveLogin(argv.username);
=======
    if (session == null) {
        session = await DoInteractiveLogin(videoUrls[0], argv.username);
    }

>>>>>>> tokencache
```

Are we now expecting one or two arguments to DoInteractiveLogin?
If you're sure about it feel free to resolve this conflict yourself and merge.

**Comment by snobu** [comment]
Nevermind, got off my high horse and looked at the actual code :) Resolved. Merging now.

**Pull request merged**